### PR TITLE
doc: generate ordered API list for metrics doc

### PR DESCRIPTION
**Pull request by rahulguptajss**

### CODE REVIEW

File: cmd/tools/generate/counter.go [hunk 1]
```diff
@@ -665,7 +665,9 @@ var reRemove = regexp.MustCompile(`NFSv\d+\.\d+`)
 
 func mergeCounters(restCounters map[string]Counter, zapiCounters map[string]Counter) map[string]Counter {
 	// handle special counters
-	for k, v := range restCounters {
+	restKeys := sortedKeys(restCounters)
```
Comment (cgrinds) [inline cmd/tools/generate/counter.go:668]: restKeys := maps.Keys(restCounters)

slices.Sort(restKeys)

**Review by cgrinds — CHANGES_REQUESTED** [review]

File: cmd/tools/generate/counter.go [hunk 2]
```diff
@@ -681,7 +684,10 @@ func mergeCounters(restCounters map[string]Counter, zapiCounters map[string]Coun
 		}
 	}
 
-	for k, v := range zapiCounters {
+	zapiKeys := sortedKeys(zapiCounters)
+	sort.Strings(zapiKeys)
```
Comment (cgrinds) [inline cmd/tools/generate/counter.go:688]: not needed?

File: cmd/tools/generate/counter.go [hunk 3]
```diff
@@ -717,6 +726,12 @@ func mergeCounters(restCounters map[string]Counter, zapiCounters map[string]Coun
 	return restCounters
 }
 
+func sortedKeys(m map[string]Counter) []string {
+	keys := maps.Keys(m)
+	sort.Strings(keys)
```
Comment (cgrinds) [inline cmd/tools/generate/counter.go:731]: slices.Sort is better to use since it is faster

File: docs/ontap-metrics.md [hunk 4]
```diff
@@ -1569,6 +1569,15 @@ Number of discarded frames.
 | ZAPI | `perf-object-get-instances fcp_port` | `discarded_frames_count`<br><span class="key">Unit:</span> none<br><span class="key">Type:</span> delta<br><span class="key">Base:</span>  | conf/zapiperf/cdot/9.8.0/fcp.yaml | 
 
 
+### fcp_fabric_connected_speed
+
+
+
```
Comment (Hardikl) [inline docs/ontap-metrics.md:1575]: empty description?

**Review by Hardikl — DISMISSED** [review]

**Review by Hardikl — APPROVED** [review]

**Review by cgrinds — APPROVED** [review]

**Pull request merged**